micromixin-annotations/src/main/java/org/spongepowered/asm/mixin/Mutable.java

```diff
@@ -1,9 +1,17 @@
 package org.spongepowered.asm.mixin;
 
 import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-// TODO: Implement for methods
-@Target({ElementType.FIELD})
+/**
+ * {@link Mutable} can only be applied to {@link Shadow}ed fields or methods.
```

Comment:
A minor nitpick, but the linking conventions around here would mean that you use `{@link Shadow shadowed}`. Although this is more of a personal choice I suppose